src/test/run-cli-tests

```diff
@@ -40,29 +61,30 @@ mkdir -p "$FAKE_HOME"
 
 # cram doesn't like seeing the same foo.t basename twice on the same
 # run, so run it once per directory
-FAILED=0
-FAILEDTOOLS=""
+tools=()
```

Comment:
unlike Bash, POSIX shell does not support array.

Reply:
Didin't caught that. So I switch to bash to keep this feature. Many script files are running bash, so let's do the same here.